README.md

```diff
@@ -57,8 +46,257 @@ Express has extensive support for template rendering and can load and leverage H
 
   Handlebars will allow us to avoid duplication by creating template views. In fact Handlebars will allow us to delete all those `.html` files.
 
-  Let's code-along together!
+### Install handlebars
+
+ 1. In the tasks directory run the follwing code:
+
+    ```bash
+    npm i --save express-handlebars
+    ```
+
+### Add handlebars to your project
+
+1. Require express-handlebars in app.js:
+
+    ```js
+    const exphbs = require('express-handlebars')
+    ```
+
+2. We now need to tell express the location of the ```views``` folder and the views engine that we want to use:
+    ```js
+    app.set('views', path.join(__dirname, 'views'));
+    app.set('view engine', 'hbs');
+    ```
+
+3. We use ```app.engine``` to tell express to use the view engine (express-handlebars) to render hbs files. We are also configuring express-handlebars by giving the various directories that the engine will search for, and the default layout.
```

Comment:
I do not think we have described the word `render`?

Reply:
Added explanation of render first time it is mentioned https://github.com/foundersandcoders/express-handlebars-workshop/pull/11/commits/a89cb51c363c48232c5e8c300648fea0537399dc